src/MomentoCacheClient.php

```diff
@@ -17,12 +17,349 @@ public function __construct(CacheClient $client, string $cacheName)
         $this->cacheName = $cacheName;
     }
 
+    public function _compress(string $value): string
+    {
+        throw new NotImplementedException();
+    }
```

Comment:
I recommend a helper function to throw an unsupported command error, see [here](https://github.com/momentohq/momento-java-lettuce-client/blob/main/src/main/java/momento/lettuce/utils/MomentoToLettuceExceptionMapper.java#L41-L50). This should include the name of the command/method not supported. In the Lettuce wrapper client that ended up like [this](https://github.com/momentohq/momento-java-lettuce-client/blob/main/src/main/java/momento/lettuce/MomentoRedisReactiveClient.java#L160-L183).